src/styles/main.scss

```diff
@@ -1,3 +1,203 @@
-body {
+$blue-color : #00acdc;
+$secondary-color: #616070;
+$white-color: #fff;
+
+html {
+  font-family: Roboto, sans-serif;
+  color: #060b35;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.header {
+  width: 100%;
+  display: flex;
+  padding: 0 50px;
+  align-items: center;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: block;
+}
+
+.nav {
+
+  &__list {
+    display: flex;
+    list-style: none;
+  }
+
+  &__item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__link {
+    display: block;
+    text-decoration: none;
+    text-transform: uppercase;
+    color: #060b35;
+    padding: 23px 0;
+  }
+
+}
+
+.nav__link:hover {
+  color: $blue-color;
+}
+
+.is-active {
+  position: relative;
+  color: $blue-color;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  width: 37px;
+  height: 4px;
+  background: $blue-color;
+  border-radius: 8px;
+  bottom: 0;
+  left: 0;
+  right: 0;
+}
+
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+  max-width: 950px;
+  justify-content: center;
```

Comment:
comment about broken layout not fixed.
for fix it enough not set justify-content: center (by default this property `start`)
```suggestion
```
![Screenshot from 2023-04-17 21-21-50](https://user-images.githubusercontent.com/33029864/232577143-6c0cd847-0193-49b3-869f-506f449464a2.png)